packages/story-editor/src/utils/useCORSProxy.js

```diff
@@ -30,10 +30,38 @@ function useCORSProxy() {
     actions: { getProxyUrl },
   } = useAPI();
   const enableCORSProxy = useFeature('enableCORSProxy');
+
```

Comment:
This change is wholesale copied from https://github.com/google/web-stories-wp/pull/9713.

Reply:
If you add this here you also need to remove it from `useInsert`